models/intel/smartlab-sequence-modelling-0001/README.md

```diff
@@ -0,0 +1,177 @@
+# online mstcn++
+
+## Use Case and High-Level Description
+This is an online action segmentation network for 16 classes trained on Intel dataset. It is an online version of MSTCN++. The difference between online MSTCN++ and MSTCN++ is that the former accept stream video as input while the latter assume the whole video is given.
+
+For the original MSTCN++ model details see [paper](https://arxiv.org/abs/2006.09220)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| GOPs                            | 0.048915                                  |
+| MParams                         | 1.018179                                  |
+| Source framework                | PyTorch\*                                 |
+
+## Accuracy
+<table>
```

Comment:
could you please use markdown style table instead html?

Reply:
Hi, I think github markdown style don' support **merge table rows**. If I'm wrong, could you please give some example?